src/main/java/com/github/fishio/achievements/Achieve.java

```diff
@@ -70,9 +68,10 @@ else if (EnemyKillObserver.getCounter() >= 10) {
 			enemyKill.setLevel(2);
 			Log.getLogger().log(LogLevel.INFO, "Achievement gained for killing an enemy fish 10 times");
 		}
-		else if (EnemyKillObserver.getCounter() >= 3) {
```

Comment:
This is temporarily to make it easier to manually test if the implementation works. When everything works, it will be set back to 3.

<testing